testsuite/features/step_definitions/common_steps.rb

```diff
@@ -64,7 +64,7 @@
   steps %(
     When I follow "Events"
     And I follow "Pending"
-    And I wait at most 60 seconds until I do not see "#{event}" text, refreshing the page
+    And I wait at most 90 seconds until I do not see "#{event}" text, refreshing the page
```

Comment:
😆 I told you... 👯

Reply:
Yes....

Jokes apart, I'm still wondering how it can be you have to wait for one minute and a half for an event to be picked. There must be some product issue, especially that it only happens on centos systems. I did not create a bug report as it would probably not get attention, but something is fishy there.

An investigation in the log files did not find anything. Furthermore, I have not been able to reproduce when running exactly the same features. Could it be that the server is busy doing something else when we run the full BV test suite ? Or is it linked to the package we try to patch (`timezone-data`) ?

@srbarrios